### DISCUSSION TITLE
Idea: Keep model in the GPU memory

### DISCUSSION THREAD

**Discussion by linonetwo** [comfyanonymous/ComfyUI#535] [Ideas]
Seems stable difussion webui will keep model (checkpoint) in the GPU memory, so you don't need to send it again, is this correct?

And I found ComfyUI seems keep it in normal memory, and send it to GPU on each workflow execution.

Will SD webui's approach faster?

**Comment by comfyanonymous** [comment]
You can use: --highvram in ComfyUI to keep the model in GPU memory.

**Reply by linonetwo** [reply]
Basically the same

when drawing, the usage is high

![图片](https://user-images.githubusercontent.com/3746270/232806850-c7a68cf6-cbf0-4dd9-8e72-b9688eb2fec5.png)

when finished, it drop to 3.1GB, compare to 1-2GB when without `--highvram`

![图片](https://user-images.githubusercontent.com/3746270/232807025-d3471ef3-777f-42b9-8fa4-b074e79e6788.png)

**Reply by linonetwo** [reply]
Anyway, I'm not sure if this affects performance, so never mind.

**Reply by WASasquatch** [reply]
I don't believe the model is stored on the "GPU". It's stored in RAM, for quick access by the GPU when computing. Least it was common practice to move to CPU (RAM) when not actually computing.
